src/System.Runtime.Serialization.Xml/tests/DataContractSerializer.cs

```diff
@@ -2599,6 +2600,71 @@ public static void BasicRoundTripResolveDTOTypes()
         ObjectContainer deserialized = SerializeAndDeserialize(instance, expectedxmlstring, null, serializerfunc, false);
         Assert.Equal(DateTimeOffset.MaxValue, ((DTOContainer)deserialized.Data).nDTO);
     }
+
+    [ActiveIssue(12767)]
+    [Fact]
+    public static void ExtensionDataObjectTest()
+    {
+        PersonV2 p2 = new PersonV2();
+        p2.Name = "Elizabeth";
+        p2.ID = 2006;
+        DataContractSerializer ser =
```

Comment:
We can improve it later. The test doesn't work for active issues. The test need more improvement later.